CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+CODEOWNERS @dataxu/releng
+Dockerfile @dataxu/releng
+Jenkinsfile @dataxu/releng
```

Comment:
not sure why this commit was included in my PR, but it should be here anyway so I'll keep it